code_annotations/base.py

```diff
@@ -366,6 +375,7 @@ def _check_results_choices(self, annotation):
                 annotation,
                 'No choices found for "{}". Expected one of {}.'.format(token, self.config.choices[token])
             )
+        return
```

Comment:
Any reason why this is a simple `return` whereas the change directly below this is a `return None`?

Reply:
I was making this the same as a return statement earlier in this function (see line 351) for consistency.  Functionally `return` and `return None` do the same thing.

Reply:
Ah -- Could we pick one then and stick with it for consistency then?

Reply:
Changed it to `return None` for consistency.